front/gatsby/src/components/Layout/Header/Header.tsx

```diff
@@ -192,54 +40,71 @@ export function Header(): React.ReactElement {
   }, [fetchAccounts]);
 
   useEffect(() => {
+    const count = getCartItemsCount();
+    setNumberOfItemsInCart(count);
     handleLogin();
+
+    // FIXME: use store.js for window.addEventListener('storage', updateCartItems);
```

Comment:
"there's a hook for everything" https://usehooks.com/useLocalStorage/ (just fyi, not saying it's better than window.addEventListener)

Reply:
i think you would need to put all cart items under one key, aka an array inside one locastroage key inside of `n` keys. But should be doable